openshift/template.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: assisted-events-scrape
```

Comment:
any chance we can change the repo name?

Reply:
changed the repo name, and update openshift/release as well
https://github.com/openshift/release/pull/23441

Reply:
Thank you!
Let me know when you need review for the change in gitlab

Reply:
https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/28417